packages/ts-transform/src/utils/template-literal-to-css.tsx

```diff
@@ -18,9 +18,18 @@ const extractSuffix = (tail: string) => {
   if (tail[0] === '\n' || tail[0] === ';') {
     rest = tail;
   } else {
-    // Sometimes people forget to put a comma at the end.
-    // This handles that case.
-    let tailIndex = tail.indexOf(';') === -1 ? tail.indexOf('\n') : tail.indexOf(';');
+    // Sometimes people forget to put a semi-colon at the end.
+    let tailIndex;
+    if (tail.indexOf(')') !== -1) {
```

Comment:
can you leave a comment here explaining the use case its solving? (for calc, transform, etc)

Reply:
Added